packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -20,13 +22,28 @@ export const StoryPromoLi = styled.li`
     padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING_TRPL};
   }
 
+  &:first-child {
+    padding-top: 0;
+  }
+
   &:last-child {
+    padding-bottom: 0;
     border: none;
   }
 `;
 
-export const StoryPromoUl = styled.ul`
+const StyledUnorderedList = styled.ul`
   list-style-type: none;
   margin: 0;
   padding: 0;
 `;
+
+export const StoryPromoUl = ({ children, ...props }) => (
```

Comment:
Hey, soz can you do the same react component design with `StoryPromoLi` with `listitem` (i think(double check a11y spec))